movie-subdomain/src/main/kotlin/com/example/ajaxonboardingproject/application/service/MovieService.kt

```diff
@@ -1,6 +1,6 @@
-package com.example.ajaxonboardingproject.service
+package com.example.ajaxonboardingproject.application.service
```

Comment:
Lets call this class MovieInPort as it's input port to application level from infra level. and again, MovieServiceImpl can become just MovieService then

Reply:
Done! ✅